### PR TITLE
Adb subprocess tests

### PR DESCRIPTION
**Pull request by xiamaz**
Unittests for communication with the fakeinputlib adb subprocess.

### CODE REVIEW

**Comment by xiamaz** [comment]
PTAL

**Comment by xiamaz** [comment]
The linter seems to timeout. I've got no idea why.

File: app/src/main/java/com/amos/flyinn/summoner/ADBService.java [hunk 1]
```diff
@@ -26,28 +30,66 @@ public ADBService() {
...
                 @Override
                 public String encodeToString(byte[] data) {
                     return android.util.Base64.encodeToString(data, 16);
                 }
             });
-            AdbConnection connection = AdbConnection.create(socket, crypto);
+            connection = AdbConnection.create(socket, crypto);
             connection.connect();
-            AdbStream stream = connection.open(workIntent.getStringExtra("cmd"));
+        } catch (NoSuchAlgorithmException err) {
```
Comment (lbb) [inline app/src/main/java/com/amos/flyinn/summoner/ADBService.java]: Why are we catching different exceptions? We don't handle them differently?

**Review by lbb — DISMISSED** [review]

**Comment by xiamaz** [comment]
Travis build failure should be fixed now

**Comment by xiamaz** [comment]
Please approve if travis is passing

**Review by lbb — APPROVED** [review]

**Pull request merged**